### ISSUE TITLE
"Post on Behalf of Users" Review Adjustments - I

### ISSUE THREAD

**Issue by 0x4007** [ubiquity/ubiquibot-telegram#23]
> There's a handful of small adjustments I'd like that we can roll into new task(s) after merging in the current pull request:

- [x] Issue titles should be Title Case and no punctuation at the end
- [x] No more block quote for the message extracted from telegram (we ignore blockquotes for comment incentives. Just write with normal text.
- [x] "[ View Conversation Context ]" to "[ View on Telegram ]"

_Originally posted by @pavlovcik in https://github.com/ubiquity/telegram-ubiquibot/pull/22#pullrequestreview-1618850334_

**Comment by seprintour** [comment]
> Do you have any updates @seprintour? If you would like to release the bounty back to the DevPool, please comment `/stop` Last activity time: Sun Sep 10 2023 18:35:25 GMT+0000 (Coordinated Universal Time)

Highly dependent on #25

**Comment by 0x4007** [comment]
> #### Conversation Rewards
> ### [ **seprintour: [ CLAIM 1.4 WXDAI ]** ](https://pay.ubq.fi?claim=eyJwZXJtaXQiOnsicGVybWl0dGVkIjp7InRva2VuIjoiMHhlOTFEMTUzRTBiNDE1MThBMkNlOERkM0Q3OTQ0RmE4NjM0NjNhOTdkIiwiYW1vdW50IjoiMTQwMDAwMDAwMDAwMDAwMDAwMCJ9LCJub25jZSI6IjUwNzgyNDY2NzI1MjgyNDM4NzY2NTQ3NzE2NTc3MjU1NzMxNjc4MTA2MjkyMzc3MzM5NTA0ODY2MTM2OTEyMTg4NjQ2MjYxODA3MDEzIiwiZGVhZGxpbmUiOiIxMTU3OTIwODkyMzczMTYxOTU0MjM1NzA5ODUwMDg2ODc5MDc4NTMyNjk5ODQ2NjU2NDA1NjQwMzk0NTc1ODQwMDc5MTMxMjk2Mzk5MzUifSwidHJhbnNmZXJEZXRhaWxzIjp7InRvIjoiMHgzNjIzMzM4MDQ2YjEwMWVjRWM3NDFEZTlDMzU5NENDMjE3NmYzOUU1IiwicmVxdWVzdGVkQW1vdW50IjoiMTQwMDAwMDAwMDAwMDAwMDAwMCJ9LCJvd25lciI6IjB4Zjg3Y2E0NTgzQzc5MjIxMmU1MjcyMGQxMjdFN0UwQTM4QjgxOGFEMSIsInNpZ25hdHVyZSI6IjB4M2ZhMTYxMjU3NDk1ZDU5OGRkMmM2YjMxZTIwN2JmZDNiY2MxYjkyZDZmNzc1MGFiMmQ4NGVjMDhhMmVjNWQxMDRlMWUwN2FjYzhmMDlkYWJjYzQxNzFkNWZmZGIwN2YwODU1NjgzMjMwOGQxZTY2ZWUwYmZmMjAwMjM5MjUxZWQxYiJ9&network=100)

I really don't understand this unpredictable behavior. It should not provide you with comment incentives as the assignee.

@0xcodercrane @wannacfuture @whilefoo rfc

**Comment by seprintour** [comment]
> I really don't understand this unpredictable behavior. It should not provide you with comment incentives as the assignee.

Yea, i think the feature to disable assignee from comment incentives was just added but its not in production yet

**Comment by 0x4007** [comment]
I thought the original implementation handled that already. I could be wrong though.

**Comment by seprintour** [comment]
> I thought the original implementation handled that already. I could be wrong though.

The original implementation used `assignee` instead of `assignee.login`.. I see this error a lot, it easy to miss that assignee is an object and not a string

**Comment by wannacfuture** [comment]
> I thought the original implementation handled that already. I could be wrong though.

It is newly fixed. Seems it was handled wrong in the original implementation.

**Comment by cuddle-dev** [comment]
> Yea, i think the feature to disable assignee from comment incentives was just added but its not in production yet

Rolling out the recent changes to prod today.
